Scripts/Events/GunEvents.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using Godot;
+
+namespace PhotonPhighters.Scripts;
+
+public partial class GunEvents
+{
+  public partial class ShootEvent : Node
```

Comment:
Only one event, can remove the GunEvents class. Perhaps create a Events namespace?